src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -614,6 +614,9 @@ internal bool Connected
         // Returns:
         // 
         //     An IASyncResult, representing the read.
+#if !netcore50
+        override
+#endif
```

Comment:
I don't understand why we would drop 'netcore50' suport.  I don't really care what moniker we call it.  But we need to have UWP working at all times.  Will I still be able to build a UWP app and use System.Net.Sockets (or any other networking library).  And when/how will UWP be updated in order to use the newly added contract API surface. I've heard it is when "uap10.1" gets working.  But I don't understand exactly when that is and how it works.